instructure/canvas-ios
# Update offline sync selector loading state.

refs: MBL-17066
affects: Student
release note: none

test plan:
- Enter "Manage Offline Content" screen on an account with lots of courses and files.
- Check the loading state of the page: it should display a still panda with a message.

## Screenshots

<img src="https://github.com/instructure/canvas-ios/assets/72396990/ceef1619-a802-4cb4-86bb-cc890ccc3ab7" width=50% height=50%>

## Checklist
